scripts/after_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+export PATH=$PATH:/home/ubuntu/.cargo/bin
+
+APP_NAME=$(echo $APPLICATION_NAME | awk -F- '{ print $2 }')
+BRANCH=$(curl http://169.254.169.254/latest/meta-data/tags/instance/$APP_NAME)
```

Comment:
What server is this? Can we use a name here?

Reply:
169.254.169.254 is used as the endpoint for the instance metadata service. This is an internal IPv4 address, and every EC2 instance has this IP address.